test/lib.bash

```diff
@@ -84,9 +98,88 @@ function run_knative_serving_tests {
     --imagetemplate "$image_template" \
     || failed=1
   
-  rm -rf "$tmp_gopath"
   return $failed
-  )
+}
+
+function run_knative_serving_rolling_upgrade_tests {
+  logger.info "Running rolling upgrade tests"
+  local failed=0
+
+  go test -tags=preupgrade -timeout=20m ./test/upgrade \
+  --imagetemplate "$TEST_IMAGE_TEMPLATE" \
+  --kubeconfig "$KUBECONFIG" \
+  --resolvabledomain || failed=1
+
+  echo "Starting prober test"
+
+  # Make prober send requests more often compared with upstream where it is 1/second
+  sed -e 's/\(.*requestInterval =\).*/\1 200 * time.Millisecond/' -i vendor/knative.dev/pkg/test/spoof/spoof.go
```

Comment:
Why not make this change in the code of the v0.10.0 branch in openshift/knative-serving to avoid this hack?